File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int[] freq = new int[101];
        for(int num : nums){
            freq[num] += 1;
        }
        
        int sum = 0;
        for(int i = 0; i < freq.length; i++){
            if(freq[i] == 1){
                sum += i;
            }
        }
        return sum;
    }
}

File: 1848-sum-of-unique-elements/sum-of-unique-elements.py
class Solution(object):
    def sumOfUnique(self, nums):
        freq = [0] * 101  # frequency array of size 101
    
        for num in nums:
           freq[num] += 1

        total = 0
        for i in range(len(freq)):
          if freq[i] == 1:
            total += i

        return total
